package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "ad-ldap-connector",
-  "version": "6.1.8",
+  "version": "6.1.9",
   "description": "ADLDAP Federation Connector",
   "main": "server.js",
   "scripts": {
+    "lint:check": "npx eslint admin/**/*.js connector-setup/**/*.js lib/** test/**/*.js",
+    "lint:fix": "npx eslint admin/** connector-setup/** lib/** test/** --fix",
     "test": "NODE_TLS_REJECT_UNAUTHORIZED=0 mocha --timeout 50000 --reporter spec --exit",
     "snyk": "snyk test",
-    "start": "node server.js"
+    "start": "node server.js",
+    "start-local": "source .env && npm run start"
```

Comment:
This is at least an unexpected pattern to me, I've typically used the language's [dotenv](https://www.npmjs.com/package/dotenv) project to load a `.env`